cluster/store/query.go

```diff
@@ -160,3 +165,21 @@ func (st *Store) QueryTenantsShards(req *cmd.QueryRequest) ([]byte, error) {
 	}
 	return payload, nil
 }
+
+func (st *Store) QueryShardingState(req *cmd.QueryRequest) ([]byte, error) {
+	// Validate that the subcommand is the correct type
+	subCommand := cmd.QueryShardingStateRequest{}
+	if err := json.Unmarshal(req.SubCommand, &subCommand); err != nil {
+		return []byte{}, fmt.Errorf("%w: %w", errBadRequest, err)
+	}
+
+	// Read the meta class to get both the class and sharding information
```

Comment:
nitpick(non-blocker): comment isn't related to the change under the hood

Reply:
i blame copy/paste :D